doc/code/qml_drawer.rst

```diff
@@ -42,6 +42,12 @@ The following images are generated via the code:
     qml.drawer.use_style(style)
     fig, ax = qml.draw_mpl(circuit)(1.2345, 1.2345)
 
+Note that the shown ``pennylane`` style can be replicated exactly by having the
```

Comment:
In the example above this, it would actually be handy for `style` to already be one of the possible strings to put, it is not clear for a beginner from that page what they should put (I was able to guess but I dont think newbies would)

Reply:
Ah, I see what you mean. I'll add a sentence to explain.